yrs/src/update.rs

```diff
@@ -254,20 +257,16 @@ impl Update {
                     stack_head = stack.pop();
                 } else {
                     current_target = match current_target.take() {
-                        None => None,
```

Comment:
This is 2 of 2 bugs detected by David: in some edge cases current_target could be null. We should proceed to a next client block list.